_summaries/Rae2019compressive_2.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: summary
+title: Summary
+paper: Rae2019compressive_2
+# Please fill out info below
+author: specfazhou
+score: 8/10
+---
+
+## core idea
+In this article, the authors proposed Compressive Transformer, a variant of Transformer, which achieved the-state-of-art resutls on character-based language modelligng and word-level language modelling.
+Compressive Transformer also performed very well on the book-level language modelling. The Compressive Transformer, neither like Transformer that computationally expensive, nor like TransformerXL that discard old enough activations, it compress past activations into compressed memory.
```

Comment:
I don't think you can say that Compressive Transformer remembers "all past activations".

Reply:
Can you tell me what part of the paper mentions compressed memory will be discarded?